fedra.sh

```diff
@@ -0,0 +1,60 @@
+package: FEDRA
+version: rev1523
+tag: rev1523
+
+source: https://github.com/antonioiuliano2/fedra
+
+requires:
+  - GCC-Toolchain
+  - ROOT
+
+--- 
+#!/bin/bash 
+rsync -a $SOURCEDIR/* $BUILDDIR
+
+export FEDRA_INSTALL_DIR=$INSTALLROOT
+
+chmod u+x install.sh
+./install.sh
```

Comment:
You could probably just use `sh install.sh` (or `bash install.sh`) here. Alternatively, you could `source install.sh`, although that would risk messing up environment variables for the rest of the recipe.